ui-react/packages/atlasmap-ui/src/AtlasmapUI/MappingDetails/MappingField.tsx

```diff
@@ -68,92 +69,97 @@ export const MappingField: FunctionComponent<IMappingFieldProps> = ({
-                  type={'number'}
-                  value={index}
-                  id={'index'}
-                  disabled={!canEditIndex}
-                  className={css(styles.indexInput)}
-                  onChange={onIndexChange}
-                />
-              </Label>
-            </Tooltip>
-          )}
-        </SplitItem>
-        <SplitItem>
-          <Stack>
-            <StackItem isFilled />
-            <StackItem>
-              <Button
-                variant={'link'}
-                onClick={onDelete}
-                className={css(styles.link)}
+                  <Tooltip
+                    position={'auto'}
+                    enableFlip={true}
+                    content={<div>{info}</div>}
+                  >
+                    <span>
+                      <InfoAltIcon className={css(styles.icon)} /> {name}
+                    </span>
+                  </Tooltip>
+                </Title>
+              </StackItem>
+              <StackItem isFilled />
+            </Stack>
+          </SplitItem>
+          <SplitItem>
+            {showIndex && (
+              <Tooltip
+                position={'auto'}
+                enableFlip={true}
+                content={
+                  <div>
+                    Edit the index for this element by selecting the arrows.
+                    Placeholders may be automatically inserted to account for
+                    any gaps in the indexing
+                  </div>
+                }
               >
-                Remove
-              </Button>
-            </StackItem>
-            <StackItem isFilled />
-          </Stack>
-        </SplitItem>
-      </Split>
-      // Show established field action transformations associated with this
-      field.
-      {Children.count(children) > 0 && (
-        <DataListContent
-          aria-label={'Field transformations'}
-          className={css(styles.dataListContent)}
-        >
-          <Title
+                <Label>
+                  #{' '}
+                  <input
+                    type={'number'}
+                    value={index}
+                    id={'index'}
+                    disabled={!canEditIndex}
+                    className={css(styles.indexInput)}
+                    onChange={onIndexChange}
+                  />
+                </Label>
+              </Tooltip>
+            )}
+          </SplitItem>
+          <SplitItem>
+            <Stack>
+              <StackItem isFilled />
+              <StackItem>
+                <Button
+                  variant={'link'}
+                  onClick={onDelete}
+                  className={css(styles.link)}
+                >
+                  Remove
+                </Button>
+              </StackItem>
+              <StackItem isFilled />
+            </Stack>
+          </SplitItem>
+        </Split>
+        // Show established field action transformations associated with this
+        field.
```

Comment:
Does this build okay?  (field not under comment)

Reply:
Wow, that's really strange. Yes, it builds okay, but I'm not entirely sure how. Fixed now.